co-log/co-log.cabal

```
@@ -97,3 +97,15 @@ executable readme
   build-tool-depends:  markdown-unlit:markdown-unlit
   ghc-options:         -Wall -pgmL markdown-unlit
   default-language:    Haskell2010
+
+test-suite property
+  build-depends:        base >= 4.9 && < 5
+                      , hedgehog
```

Stricter constraints for the library
```suggestion
                      , hedgehog ^>= 0.6
```